packages/docs/pages/toggle.tsx

```diff
@@ -0,0 +1,124 @@
+import { H1, Panel, Text, Toggle } from '@bigcommerce/big-design';
+import { VisibilityIcon, VisibilityOffIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { Code, CodePreview, ContentRoutingTabs, GuidelinesTable, List } from '../components';
+import { ToggleItemPropTable, TogglePropTable } from '../PropTables';
+
+const TabsPage = () => {
+  return (
+    <>
+      <H1>Toggle</H1>
+
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Toggle</Code> buttons allow users to switch between alternative states or
+          modes of the same entity. Only one state is shown at a time.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>
+            In forms to let users switch between different options of the setting with dependent
+            subsettings.
+          </List.Item>
+        </List>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <CodePreview>
+                  {/* jsx-to-string:start */}
+                  {function Example() {
+                    const [activeTab, setActiveTab] = useState('product');
+
+                    const items = [
+                      { value: 'none', label: 'None' },
+                      { value: 'product', label: 'Product' },
+                      { value: 'varinat', label: 'Varinat' },
+                    ];
+
+                    return (
+                      <Toggle
+                        id="toggle-example"
+                        items={items}
+                        onChange={setActiveTab}
+                        value={activeTab}
+                      />
+                    );
+                  }}
+                  {/* jsx-to-string:end */}
+                </CodePreview>
+              ),
+            },
+            {
+              id: 'icons',
+              title: 'Icons',
+              render: () => (
+                <CodePreview>
+                  {/* jsx-to-string:start */}
+                  {function Example() {
+                    const [activeTab, setActiveTab] = useState('invisible');
+
+                    const items = [
+                      { icon: <VisibilityIcon />, value: 'visible' },
+                      { icon: <VisibilityOffIcon />, value: 'invisible' },
```

Comment:
We also might need some `aria-labels` on these icons for accessibility. Just make sure it works as expected.

Reply:
✅